app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Look at the subjects you could teach
+description: |-
+  Look at the subjects you could teach and choose what teacher training course you would like to do.
```

Comment:
@caitlin-reidy can we change look as some people may have accessibility needs.

Also do we want to change the url if we're saying it's not all subjects to just subjects?